src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.ts

```diff
@@ -28,6 +46,10 @@ export class UBSSubmitOrderComponent implements OnInit, OnDestroy {
     this.destroy.unsubscribe();
   }
 
+  getFormValues(): boolean {
+    return true;
+  }
```

Comment:
what the point of this ?

Reply:
the function "getFormValues" effect on the warning window appearing. It does return a boolean. In this particular case (Step 3), I always need to show the warning window.